### PR TITLE
Contacts rework

### PR DESCRIPTION
**Pull request by pvh**
With the Omnibar cleaned up, I've started on a rework of the contacts view. This is to clear the way for an ambient history view that will let you see when your contacts (and devices!) are up-to-date with their peers.

The first manifestation of this is the start of an "Authors" section in the titlebar. Later, this is where we imagine the history will be shown with contributors positioned along the X axis at their last known position.

As part of this work, I've moved the self-profile editor into the right-hand "people" button, and I've removed the authors tab from in there as well.

Still TODO
 - [x] what if authors is too wide to fit up there? how should that work?
 - [x] focusing the search box when clicking on the map doesn't seem to work

### CODE REVIEW

**Comment by pvh** [comment]
So now on this patch I've experimented with putting the omnibox under a map icon'd drop-drown. Not sure how much I like this, so we can bin it if this feels like the wrong direction. If we do go this way we could put the invitations badge onto the map icon. 

I suspect / to open is broken in this version, and there's also a bug where the first time you open the omnibox it shows no results until you press enter, after which it seems fine.

**Comment by todrobbins** [comment]
@pvh is this far enough along to test?

**Comment by pvh** [comment]
almost. hoping to get there before lunch today.

**Comment by pvh** [comment]
Okay, this is quite testable now! Feedback is welcome, both vague opinion stuff, and specific things like CSS inconsistencies or weird keyboard handling. Screen captures for bonus points.

**Comment by pvh** [comment]
I decided not to care about the too-many-authors problem. I'm sure there will be other broken things in that case anyway.

File: src/renderer/components/workspace/authors.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      boardHandle.change((board) => {
+        if (!board.authorIds) {
+          board.authorIds = []
+        }
+        board.authorIds.push(selfId)
+      })
+    }
+  }
+
+  render = () => {
```
[the remark this answers is not in the document]
Comment (pvh) [reply src/renderer/components/workspace/authors.js]: We can fix this during the TS upgrade, ya?

**Pull request merged**